articles/article-84-turing-award.md

```diff
@@ -0,0 +1,28 @@
+<!-- BEGIN TITLE -->
+# Why no Nobel prize for computer scienctists?
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Some people say it is because that Alfred Nobel’s fiancée had an affair with a mathematician, others say because he was more interested in Physics, Chemistry, Medicine, Literature, and Peace than in Mathematics, and computer science is a part of the Mathematics field. 
+
+Luckily, there is a "Nobel Prize of Computing" which is the A.M. Turing Award. It is an annual prize given by the Association for Computing Machinery (ACM) to an individual who makes (a) major contribution(s) to the computer field. The award is named after Alan Mathison Turing, who was a mathematician and computer scientist. The first award was at 1966 and went to Alan J. Perlis for his achievements in advanced computer programming techniques and compiler construction. And, the last one was at 2018, and went to Yoshua Bengio, Geoffrey Hintonor, and Yann LeCun together, for their improvements in the deep neural networks. 
```

Comment:
- you don't need the parenthesis at all since this is not a legal document and the plural case covers the singular. So just say .. who makes major contributions to the computer field. 
- In 1966, Alan J. Perlis received the first Turing award for his chievements in advanced computer programming techniques and compiler construction. 
- technically the last one did not get sent out yet since the world has not yet ended (as far as I can tell). Most recently, in 2018, Yoshua Bengio, Geoffrey Hintonor, and Yann LeCun received the award for improving deep neural networks. (the issue with this sentence is it gets frequently outdated)